app/assets/javascripts/custom.js

```diff
@@ -0,0 +1,8 @@
+function burgerMenu() {
+  var x = document.getElementById("burgerNav");
```

Comment:
What does `x` mean?

Reply:
Admittedly i took this from W3Schools I think. x is the burgerNav div which links to the css so that a responsive drop down burger menu shows in mobile view. Not sure how that should be changed?